test/storybook.test.js

```diff
@@ -156,12 +156,12 @@ describe('percy storybook', () => {
 
     await expectAsync(storybook(['http://localhost:8000']))
     // message contains the client stack trace
-      .toBeRejectedWithError(/^Story Error\n.*\/iframe\.html.*$/s);
+      .toBeRejectedWithError(/^Story Errored\n.*\/iframe\.html.*$/s);
```

Comment:
Word change ? Is this same for v6 and v7

Reply:
This is a custom error that we are throwing.
Earlier in the test itself, we were mocking the error to be thrown, since `storyErrored` event was caught, but we `reject` the promise and passed nothing to it, so it returned undefined and fell back to the mocked error.

Reply:
Refactored error throwing logic, so as to not update tests.